### PR TITLE
Add reviewing info

### PR DESCRIPTION
**Pull request by SaschaMann**
For context, watch the weekly call from today.

### CODE REVIEW

File: README.md
Comment (SleeplessByte) [inline README.md:16]: This list has items which are not of equal "list-indentness" in terms of information. I don't know how to solve, but right now it seems wrong 😅

**Review by SleeplessByte — CHANGES_REQUESTED** [review]
See inline

File: README.md
Comment (BethanyG) [reply README.md:16]: How about... Might be too much, but nested does seem a bit better for which points are "main" vs "follow-on"

-----

Pull Requests to this repo require three approving reviews to be merged.

- If you think that a PR should not be merged, either as is or in general, please leave a review **requesting changes**.
- Please also use the **Request Changes** button if you are unsure about how a concern of yours could be resolved.
    - This will block the PR from being merged until the change request has been addressed / overruled by an admin.

    - It ensures that your thoughts will be heard and can be addressed rather than being overruled by a majority.
    - When a discussion goes stale or there are several opposing views and a compromise cannot be reached, ping [@ErikSchierboom](https://github.com/ErikSchierboom/) to resolve the discussion.

Comment (SaschaMann) [reply README.md:16]: > When a discussion goes stale or there are several opposing views and a compromise cannot be reached, ping @ErikSchierboom to resolve the discussion.

This point is not linked to change requests, imo, so it should be on the higher level. Otherwise that change would be fine with me.

**Review by SleeplessByte — APPROVED** [review]

**Comment by SleeplessByte** [comment]
Can you hear me now GIF

I feel ~~hurt~~ heard.

**Review by iHiD — APPROVED** [review]
Thanks for this :)

File: README.md
Comment (iHiD) [inline README.md:17]: Let's make it the maintainers-admin group to futureproof having other people in that alongside Erik, and also so things get to me if he's on holiday etc. I don't think there's a great link for that, so I've removed it, but I have no problem with some link being added back if appropriate.

```suggestion
- When a discussion goes stale or there are several opposing views and a compromise cannot be reached, ping @exercism/maintainers-admin to resolve the discussion.
```

Comment (SaschaMann) [reply README.md]: I've added a link to the team page

**Review by BethanyG — APPROVED** [review]
I like it.  😄

**Pull request merged**
